topics/backbonejs/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: backbone, backbonejs
```

Comment:
since `backbonejs` is already the name of the topic, you only need the alias of `backbone` here

```suggestion
aliases: backbone
```